fiboa_cli/datasets/pt.py

```diff
@@ -1,77 +1,48 @@
-from ..convert_utils import convert as convert_
-from .commons.admin import add_admin
-
-SOURCES = "https://www.ifap.pt/isip/ows/resources/2023/Continente.gpkg"
-#SOURCES = "https://www.ifap.pt/isip/ows/resources/2022/2022.zip"
-LAYER_FILTER = lambda layer, uri: layer.startswith("Culturas_")
-
-ID = "pt"
-TITLE = "Field boundaries for Portugal"
-SHORT_NAME = "Portugal"
-DESCRIPTION = "Open field boundaries (identificação de parcelas) from Portugal"
-PROVIDERS = [
-    {
-        "name": "IPAP - Instituto de Financiamento da Agricultura e Pescas",
-        "url": "https://www.ifap.pt/isip/ows/",
-        "roles": ["producer", "licensor"]
+from ..convert_utils import BaseConverter
+
+class PTConverter(BaseConverter):
+    id = "pt"
+    title = "Field boundaries for Portugal"
+    short_name = "Portugal"
+    description = "Open field boundaries (identificação de parcelas) from Portugal"
+    sources = "https://www.ifap.pt/isip/ows/resources/2023/Continente.gpkg"
+    def layer_filter(self, layer, uri):
+        return layer.startswith("Culturas_")
+
+    providers = [
+        {
+            "name": "IPAP - Instituto de Financiamento da Agricultura e Pescas",
+            "url": "https://www.ifap.pt/isip/ows/",
+            "roles": ["producer", "licensor"]
+        }
+    ]
+    attribution = None
+    license = {"title": "No conditions apply", "href": "https://inspire.ec.europa.eu/metadata-codelist/ConditionsApplyingToAccessAndUse/noConditionsApply", "type": "text/html", "rel": "license"}
```

Comment:
```suggestion
    license = {
        "title": "No conditions apply",
        "href": "https://inspire.ec.europa.eu/metadata-codelist/ConditionsApplyingToAccessAndUse/noConditionsApply",
        "type": "text/html",
        "rel": "license"
    }
```

Reply:
I will create a separate pull request for a single deterministic formatter, so we'll never have to discuss styling in PRs again...